CHANGELOG.md

```diff
@@ -158,6 +158,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [#541](https://github.com/teambit/bit/issues/541) add package manager config to bit.json
 - support saving dists files on a pre-configured directory relative to consumer root
 - support `bit show --compare` with json format
+- add new login feature to bitsrc
```

Comment:
- [ ] why is it under 0.12.0 and not under unreleased?
- [ ] introduce a new command `bit login` for authenticating to bitsrc